### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review the home page of my quiz site

It has the following features

1. nav : links to categories and rules page
2. main: with introduction part and different categories card along with Call to action to the categories pagr
3. Footer: links to the social websites

image

image

image

image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    font-family: 'Inter', sans-serif;
+    font-family: 'Mukta', sans-serif;
+    background-image: url("./images/blue-copy-space-digital-background_23-2148821698.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+   
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+/* background-color: #e056fd;
```
Comment (pnchinmay) [inline style.css]: Remove comments

File: style.css [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto;
```
Comment (pnchinmay) [inline style.css]: Use same indentation everywhere
Ideally, 2 spaces in html/css acc. to Google Style Guide

**Review by pnchinmay — APPROVED** [review]
Left few suggestions
Everything else looks good :100:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+
```
Comment (khushijohri2001) [inline index.html:8]: Remove extra spaces here. It will make your code look clean and improves readability

File: index.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+          </p>
+        </div>
+        <div>
+          <div class="card-actions">
+            <button class="card-btn"><a href="./play-quiz/play.html"> Play-Now </a></button>
+            <button class="card-btn"><a href="./rules/rules.html"> Read rules </a></button>
+    
+          </div>
+        </div>
+
```
Comment (khushijohri2001) [inline index.html]: Too much extra space. Please remove it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+ </div>
+
+
+</div>
+<a href="./categories/categories.html" class="see-more">Play top quizes of characters from across the world, click here</a>
+
+</div>
+
```
Comment (khushijohri2001) [inline index.html:155]: Here too

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**